requirements.in

```diff
@@ -12,3 +12,5 @@ cryptography < 3.5
 requests
 koji
 PyGObject
+reflink
+pytest-runner<=5.3.1
```

Comment:
Shouldn't pytest-runner be in requirements-build.in? AFAIK it's a build dependency of reflink

Reply:
I've tried that script for finding build deps and it didn't find it  and according  https://gitlab.com/rubdos/pyreflink/-/blob/master/setup.py it is setup dependency ....

Reply:
`setup_requires` = build dependency
`install_requires` = runtime dependency

Confusing, I know. And turns out setup_requires is deprecated now, but still usable https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#build-system-requirement

Not sure why the script doesn't find it :thinking: 
But pip-compile doesn't find it either, which means it is not a runtime dependency

Reply:
I think I've tried to put it into build deps but it wasn't working

Reply:
moved to build deps  and it is still working

Reply:
moved to build deps  and it is still working